great_expectations/expectations/metrics/column_map_metrics/column_values_match_json_schema.py

```diff
@@ -53,6 +57,8 @@ def matches_json_schema(val):
             except:
                 raise
 
-        matches_json_schema_udf = F.udf(matches_json_schema, sparktypes.BooleanType())
+        matches_json_schema_udf = F.udf(
+            lambda val: matches_json_schema(val=val), sparktypes.BooleanType()
+        )
 
         return matches_json_schema_udf(column)
```

Comment:
Why the `lambda`? What's the purpose of this and what's the return value here? I know we don't do it consistently but I've to see type hinting/documentation for this methods.

Reply:
@cdkini Using lambda makes it adhere more formally to the UDF API -- cleaner/clearer.  This (previously existing user-defined function) handles the logic of this metric.  The capability does not exist as part of standard `Spark` API, hence, Spark provides UDF capability to compensate.  Thanks.

Reply:
Will a user of this code be familiar with the UDF API? Is there any ambiguity? If so, please leave a comment that describes this logic 🙇🏽